glusterd2/main.go

```diff
@@ -119,6 +121,29 @@ func main() {
 		log.WithError(err).Fatal("Failed to generate local auth token")
 	}
 
+	// Get the Jaeger endpoints from config file
+	jaegerEndpoint := config.GetString("jaegerendpoint")
+	jaegerAgentEndpoint := config.GetString("jaegeragentendpoint")
+
+	// Create the Opencensus Jaeger exporter endpoints to log traces and spans
+	exporter, err := jaeger.NewExporter(jaeger.Options{
+		Endpoint:      jaegerEndpoint,
+		AgentEndpoint: jaegerAgentEndpoint,
+		ServiceName:   gdctx.HostName,
+	})
+	if err != nil {
+		log.WithFields(log.Fields{
+			"error": err.Error(),
```

Comment:
@sseshasa IMO: the field should have a component name, object name etc, to make debugging easy,   error message must be provided in the WithError() or Warning section.

Reply:
@prashanthpai Can you suggest the best way to log error here.

Reply:
The consistent way to log errors is to use `log.WithError(err)`. Chain it with `WithFields()` for any other context that needs to be added.